ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -282,4 +305,11 @@ hsConDeclsBinders cons
              -> [LFieldOcc GhcPs]
     get_flds flds = concatMap (cd_fld_names . unLoc) (unLoc flds)
 
-
+getLocalBindings :: RefMap Type -> LHsDecl GhcPs -> [Name]
+getLocalBindings refmap (L (locA -> (RealSrcSpan l _)) _) =
+  nubOrdOn getOccFS . filter isVarName . map fst $ locals
+  where
+    locals = getFuzzyScope (bindings refmap) start end
```

Comment:
This doesn't recurse, right? So presumably this won't handle something like:
```
f = ...
  where
     g = ...
       where
          h = ...
```
?

Worth adding as a test case even if we don't support it, but also maybe we just can support it?

Reply:
It actually does recurse, I've just pushed an update that adds this and the other test case below